test/collectionUtils/PrimitiveParentPredicate.test.ts

```diff
@@ -0,0 +1,36 @@
+import "jest-extended";
+import PrimitiveParentPredicate from "../../src/collectionUtils/PrimitiveParentPredicate";
+import DataType from "../../src/model/dataTypes/DataType";
+import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
+import EntityDefinitionBuilder from "../model/dataTypes/EntityDefinitionBuilder";
+
+describe("PrimitiveParentPredicate", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)